frontend/src/components/QuickActions.tsx

```
@@ -0,0 +1,51 @@
+import { Map, Bell, FileText, BarChart3, ChevronRight } from 'lucide-react';
```

Biome (2.3.13)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)